src/other/Table.js

```diff
@@ -66,6 +68,7 @@
     Table.prototype.publish("columnPatterns", [], "array", "Array of formatting rules for each column", null, { tags: ["Basic"], optional: true });
     Table.prototype.publish("stringAlign", "left", "set", "Array of alignment positions for strings", ["left","right","center"], { tags: ["Basic"], optional: true });
     Table.prototype.publish("numberAlign", "right", "set", "Array of alignment positions for numbers", ["left","right","center"], { tags: ["Basic"], optional: true });
+    Table.prototype.publish("childWidgets", [], "widgetArray", "Array of child widgets", null, { tags: ["Basic"], optional: true });
```

Comment:
(I am sure I will see further down, but I don't see why this would be needed, since the widgets are coming in via the "data" method)?